apps/judicial-system/web/src/routes/Prosecutor/RestrictionCase/StepThree/StepThreeForm.tsx

```diff
@@ -121,26 +128,55 @@ const StepThreeForm: React.FC<Props> = (props) => {
                 blueBox={false}
               />
             </Box>
-            {workingCase.type === CaseType.CUSTODY && (
-              <Checkbox
-                name="isIsolation"
-                label={formatMessage(rcDemands.sections.demands.isolation)}
-                tooltip={formatMessage(rcDemands.sections.demands.tooltip)}
-                checked={workingCase.requestedCustodyRestrictions?.includes(
-                  CaseCustodyRestrictions.ISOLATION,
-                )}
-                onChange={() =>
-                  setCheckboxAndSendToServer(
-                    'requestedCustodyRestrictions',
-                    'ISOLATION',
-                    workingCase,
-                    setWorkingCase,
-                    updateCase,
-                  )
-                }
-                large
-                filled
-              />
+            {workingCase.type !== CaseType.TRAVEL_BAN && (
+              <GridRow>
+                <GridColumn span="1/2">
+                  <Checkbox
+                    name="isIsolation"
+                    label={formatMessage(rcDemands.sections.demands.isolation)}
+                    tooltip={formatMessage(rcDemands.sections.demands.tooltip)}
+                    checked={workingCase.requestedCustodyRestrictions?.includes(
+                      CaseCustodyRestrictions.ISOLATION,
+                    )}
+                    onChange={() =>
+                      setCheckboxAndSendToServer(
+                        'requestedCustodyRestrictions',
+                        'ISOLATION',
+                        workingCase,
+                        setWorkingCase,
+                        updateCase,
+                      )
+                    }
+                    large
+                    filled
+                  />
+                </GridColumn>
+                <GridColumn span="1/2">
```

Comment:
I wonder if it should be possible to change the case type in extension cases. Has that been discussed?